packages/fxa-settings/src/lib/auth-errors/auth-errors.ts

```diff
@@ -8,7 +8,7 @@ export type AuthUiError = AuthServerError;
 
 const UNEXPECTED_ERROR_MESSAGE = 'Unexpected error';
 const EXPIRED_VERIFICATION_ERROR_MESSAGE =
-  'The link you clicked to verify your email is expired.';
```

Comment:
These errors seemed to me to be user facing, but it isn't clear to me how they're translated so I might be missing a piece